src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -1100,21 +1114,100 @@ subroutine decomp_rates_mimics(bounds, num_soilc, filter_soilc, &
       mimics_cn_r = params_inst%mimics_cn_r
       mimics_cn_k = params_inst%mimics_cn_k
 
+      ! Local column-level ligninNratioAvg because
+      ! not available when use_fates = .true.
+      ! Also get FATES copy of annsum_npp for FATES cases
+      if (use_fates) then
```

Comment:
recommend using a label here: a la   "fates_if:  if (use_fates) then"

Reply:
Did do for lines 1118, 1119 where the if statements are long:

```
fates_if: if (use_fates) then
         lch4_if: if (use_lch4) then
```

True, didn't do elsewhere.